clusterloader2/testing/l4lb/config.yaml

```diff
@@ -87,6 +90,12 @@ steps:
       objectTemplatePath: service.yaml
     - basename: lb-dep
       objectTemplatePath: dep.yaml
+- name: Sleep
```

Comment:
As discussed offline, right now we can have sleep, but in the future we would like to have something like "WaitForServices" so we don't need to have this sleep.